tests/E2E Tests/SimulateOidc/Contollers/MetadataController.cs

```diff
@@ -34,7 +34,7 @@ public IActionResult  OpenIdConnectConfiguration()
         }
 
         [HttpGet("/v2.0/.well-known/keys.json")]
-        public async Task<IActionResult> Keys()
+        public async Task<IActionResult> KeysAsync()
         {
             HttpClient client = new HttpClient();
             string keysDocument = await client.GetStringAsync("https://login.microsoftonline.com/common/discovery/v2.0/keys").ConfigureAwait(false);
```

Comment:
```suggestion
            string keysDocument = await client.GetStringAsync("https://login.microsoftonline.com/common/discovery/v2.0/keys");
```

Reply:
The rule that you linked says it only affects methods marked with [Test] and [Theory]. This one isn't removed in main, so I'll leave it to avoid unnecessary changes